File: Day10/Solution.java
public class Solution {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int max = 0;
        int count = 0;

        while(n>0)
            {
                if(n%2==1)
                    count++;
                else
                    count = 0;
                if(max<count)
                    max = count;
            n=n/2;
        }
        System.out.println(max);

    }
}

File: Day10/Solution.py
#!/bin/python

import sys

def max(a,b):
    return a if a>b else b

n = int(input().strip())

max_num = 0
count = 0

while n:
    while n&1:
        count += 1
        n>>=1
    max_num = max(count, max_num)
    if not n&1:
        count = 0
        n>>=1

print(max_num)
